server/clip_server/model/fast_tokenizer.py

```diff
@@ -0,0 +1,24 @@
+import os
+from functools import lru_cache
+from rust_tokenizers import PyOpenAiGptTokenizer
+from clip_server.helper import __resources_path__
+
+
+@lru_cache()
+def default_vocab():
+    return os.path.join(__resources_path__, 'vocab.json')
+
+
+@lru_cache()
+def default_merges():
+    return os.path.join(__resources_path__, 'merges.txt')
```

Comment:
Can we generate the `merges` and `vocab` content dynamically from `bpe_simple_vocab_16e6.txt.gz`? It would be a nightmare to maintain two vocabs.

Reply:
ok, will do